### DISCUSSION TITLE
What size of genome fraction can be considered low？

### DISCUSSION THREAD

**Discussion by fanqiedantang** [ablab/quast#264] [Q&A]
Hi，
     I am studying the issue of genomic chimeric contamination. When a MAG is composed of multiple reference genomes, each with a different genome fraction. But for those reference genomes with a very small proportion (such as 0.0001%), I don't think they are the source of chimerism. How to determine the minimum genome fraction?
thank you for any help！

**Comment by alexeigurevich** [comment]
Hi @fanqiedantang ,
I converted your issue into a discussion, which is more appropriate for your query. Hopefully, someone from the QUAST/genomics community will share their experience.

In my not-very-informative opinion, there is no good answer, i.e., some universal "magic" threshold. The fraction of 0.00001% definitely look like an artifact, I would say that everything below 1% also looks like this for me.
Also, a MAG (in contrast to just _metagenome assembly_) should normally consist of a single genome, maybe with some small 5-10% contamination. Usually, [checkM](https://ecogenomics.github.io/CheckM/) is used to estimate it.
